### PR TITLE
Remove cobra from cli-files

### PR DESCRIPTION
**Pull request by illicitonion**
Cobra seems to be more confusing than it is useful.

### CODE REVIEW

File: projects/cli-files/README.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ Timebox: 4 days
 
 ## Objectives:
 
-- Install and use [cobra][cobra]
 - Use go build/go install/go get etc
```
Comment (SallyMcGrath) [inline projects/cli-files/README.md:13]: - Make a go module
- Describe the basics of a process and process lifecycle
- Accept arguments on the CLI

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Update LO pls

File: projects/cli-files/README.md (hunk 1 above)
Comment (illicitonion) [reply projects/cli-files/README.md:13]: Done!

**Review by SallyMcGrath — APPROVED** [review]
LGTM

**Pull request merged**